docs/guide/known-limitations.md

```diff
@@ -34,3 +33,86 @@ you can't compare the arguments in a formula like this:
 * SUBTOTAL function does not ignore nested subtotals.
 * CHISQ.INV, CHISQ.INV.RT, CHISQ.DIST.RT, CHIDIST, CHIINV and CHISQ.DIST (CHISQ.DIST in CDF mode): Running time grows linearly with the value of the second parameter, degrees_of_freedom (slow for values>1e7).
 * GAMMA.DIST, GAMMA.INV, GAMMADIST, GAMMAINV (GAMMA.DIST and GAMMADIST in CDF mode): Running time grows linearly with the value of the second parameter, alpha (slow for values>1e7). 
+
+## Google Sheets and Microsoft Excel
+
+In certain situations, HyperFormula behaves slightly differently than Google Sheets or Microsoft Excel.
+
+The inconsistencies are due to:
+* Known limitations of Microsoft Excel or Google Sheets.
+* Known limitations of HyperFormula in its current development stage.
+* Inconsistencies between Microsoft Excel and Google Sheets.
+
+In most cases, the differences won't be observed by the end user, but we list them all for your reference:
```

Comment:
Well... we got few issue reports that they did noticed 😉 So I wouldn't agree on that line

Also, the first table is out of their hands. But second one may be fixed by customized implementations of listed functions. So if someone is determined enough, may overwrite our implementations with theirs. IMO worth to mention that those limitations may be removed, we just had to choose one implementation

Reply:
@wojciechczerniak Done: 69de5fb